source/features/comment-on-draft-pr-indicator.tsx

```diff
@@ -1,28 +1,37 @@
+import React from 'dom-chef';
+import InfoIcon from 'octicons-plain-react/Info';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
+import createBanner from '../github-helpers/banner.js';
 
-function addIndicator(button: HTMLElement): void {
-	const preposition = button.textContent.includes('Add') ? ' to ' : ' on ';
-	button.textContent += preposition + 'draft PR';
+function addDraftBanner(newCommentField: HTMLElement): void {
+	newCommentField.before(
+		createBanner({
+			icon: <InfoIcon className="m-0"/>,
+			classes: 'p-2 my-2 mx-md-2 text-small color-fg-muted border-0'.split(' '),
+			text: <>This is a <strong>draft PR</strong>. Make sure you would like to comment.</>,
```

Comment:
Wording suggestions welcome

Reply:
```suggestion
			text: <>This is a <strong>draft PR</strong>, it might not be ready for review.</>,
```


I'd also replace the info icon with the draft PR icon